src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -12,6 +12,8 @@
 import { loc, _ } from 'okta';
 import FactorUtil from '../../../util/FactorUtil';
 
+const { getPasswordComplexityDescription, getPasswordComplexityDescriptionForHtmlList } = FactorUtil;
```

Comment:
I had a re-export syntax instead like this - 
`export { getPasswordComplexityDescriptionForHtmlList } from '../../../util/FactorUtil';`

This seemed to be a good use-case for it as avoids an import and then an export again. Was wondering how this would be better over the other.
Ref: https://javascript.info/import-export#re-export

Reply:
That's because `FactorUtil` was convert to `export default` and re-export doesn't work.
We can refactor the `FactorUtil` later on (or on master) if you're interested.

Reply:
I had tested it locally and it was working. I'm not sure if I understand correctly..

Reply:
hmm, it doesn't work for we while I convert to ES6. I might make mistake somewhere.